testing/php7-yaml/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# vim: smartindent noexpandtab tabstop=4 shiftwidth=4
+# Maintainer: Renoir Boulanger <hello@renoirboulanger.com>
+
+pkgname=php7-yaml
+_pkgreal=yaml
+pkgver=2.0.0
+pkgrel=0
+_phpver=${pkgname#php}
+_phpver=${_phpver%%-*}
+pkgdesc="PHP${_phpver}"
+url="http://bd808.com/pecl-file_formats-yaml/"
+arch="all"
+license="MIT"
+makedepends="php${_phpver}-dev autoconf yaml-dev"
```

Comment:
Alpine style is not to use braces in parameter expansions unless required -- this also goes for the instances down below.